src/App.js

```diff
@@ -1,26 +1,43 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
-
+import { Container } from 'react-bootstrap'
+import NavBar from './components/NavBar';
+import Login from './components/Login';
+import {useSelector} from "react-redux";
+import {userInfoSelector} from "./state/user/user.selectors";
+import { ConnectedRouter } from 'connected-react-router'
+import { history } from './store'
+import {Switch, Route} from 'react-router-dom';
+import SignUp from "./components/Signup";
+import ForgotPassword from "./components/ForgotPassword";
+import Profile from "./components/Profile";
+import ChangePassword from "./components/ChangePassword";
+import Landing from "./components/Landing";
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    const user = useSelector(userInfoSelector)
+    return (
+        <React.Fragment>
```

Comment:
@nikhilrayaprolu This works but there is a new, shorter syntax you can use for declaring fragments.
It looks like empty tags: <> … …  … </>